src/components/Image.js

```diff
@@ -7,4 +9,11 @@ const Image = ({ height, width, text, src }) => {
 	);
 };
 
+Image.propTypes = {
```

Comment:
check these prop types to ensure they are of the correct type. Also, because you are checking for multiple types, consider passing in a single prop...and then shaping it. You can then destructure on the component definition